### PR TITLE
TECH-150: Add OpenAPI spec for Voucher APIs 

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
Add OpenAPI spec for Voucher APIs 

## Description
Add OpenAPI spec for Voucher APIs. Verified linked APIs and add them to repository

### CODE REVIEW

**Review by conradsp — CHANGES_REQUESTED** [review]
I think the API definitions look good. Just requesting a couple of minor changes to the information at the top of this spec file.

File: test/Payment_BB_Voucher_api_test [hunk 1]
```diff
@@ -0,0 +1,2021 @@
+{
+  'openapi': '3.0.0',
+  'info':
+    {
+      'title': 'GovStack Payment BB Voucher Management APIs for external BB',
+      'description':
+        "This schema is part of the Govstack Payments Building Block.\n",
+      'contact': { 'email': 'oscar.correia@maarifaeducation.com' },
```
[the remark this answers is not in the document]
Comment (KarolinaKopacz) [reply test/Payment_BB_Voucher_api_test:8]: Okay!
vijay.mauree@itu.int I found this email address on slack - is it correct?

[the remark this answers is not in the document]
Comment (conradsp) [reply test/Payment_BB_Voucher_api_test]: Yes, that email will work!

**Comment by KarolinaKopacz** [comment]
@conradsp  I realized that in the `api` folder, files `swagger.json` and `swagger.yaml` are empty.
If you do not mind I will add API specs to these files. In that case, do you think we should also keep API specs in the `test` folder as I did?

**Comment by rachellawson** [comment]
I seem to remember there being a complex history around the apis in payments. Not sure who would be able to give detail. Jaume?

**Comment by KarolinaKopacz** [comment]
@rachellawson 
Thank you for your comment.
Before starting working on cucumber test for Payments BB - Voucher APIs we received form Vijay Mauree the link to the swagger with correct API specs here: https://app.swaggerhub.com/apis/AKibuuka/voucher/1.0.3#/VoucherPreactivation/VoucherPreactivationPOST :)

**Review by dborowiecki — APPROVED** [review]

**Comment by KarolinaKopacz** [comment]
Hi @conradsp and @oscarcorreia43 ,
Can we merge the PR?

**Pull request merged**